### PR TITLE
fix: fixed call location of container and section components

### PR DESCRIPTION
**Pull request by Mardecera**
# Summary

- [x] Changed the call places of the section and container component.
- [x] The width of the clients section was corrected.

### CODE REVIEW

File: src/components/Clients/Clients.tsx [hunk 1]
```diff
@@ -4,37 +4,40 @@ import useIntlMessages from 'hooks/useIntlMessages'
...
-                {intl('home.clients.section.button')}
-                <TbExternalLink size={25} />
-              </ButtonLink>
-            </div>
-          </TitleAndButtonSection>
-          <Brandings />
-        </ClientsSectionContent>
-      </ClientsSectionWrapper>
-    </StyledClientsSection>
+    <SectionClients id='clients' size="lg" type="margin">
```
Comment (CrhistianT7) [inline src/components/Clients/Clients.tsx:19]: rename the component to ClientsSection

File: src/components/Clients/Clients.styles.ts [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 import { breakpoints } from 'shared/breakpoints'
 import { colors } from 'styles/colors'
+import Section from 'ui/Section/Section'
 
-export const StyledClientsSection = styled.section`
+export const SectionClients = styled(Section)`
   box-shadow: 0 0 1.25rem 0 rgba(30, 26, 26, 0.5);
   background-color: ${colors.white.main};
-  width: 100%;
+  width: calc(100% + 3rem);
```
Comment (CrhistianT7) [inline src/components/Clients/Clients.styles.ts:10]: add variable to explain better 3rem value

File: src/components/Hero/Hero.styles.ts [hunk 3]
```diff
@@ -4,15 +4,8 @@ import { breakpoints } from 'shared/breakpoints'
...
-  height: 100vh;
-  width: 100%;
-  position: relative;
-
-  @media only screen and (min-width: ${ breakpoints.xs }) {
-    height: calc(100vh - 4.5rem);
-    margin-top: 0;
-  }
+export const SectionHero = styled(Section)`
+  height: calc(100vh - 4.5rem);
```
Comment (CrhistianT7) [inline src/components/Hero/Hero.styles.ts:8]: add 4.5rem to a variable

Comment (Mardecera) [reply src/components/Hero/Hero.styles.ts:8]: Global

File: src/components/Clients/Clients.tsx (hunk 1 above)
Comment (Mardecera) [reply src/components/Clients/Clients.tsx:19]: Ready!

File: src/components/Clients/Clients.styles.ts (hunk 2 above)
Comment (Mardecera) [reply src/components/Clients/Clients.styles.ts:10]: Ready!

File: src/components/Hero/Hero.styles.ts (hunk 3 above)
Comment (Mardecera) [reply src/components/Hero/Hero.styles.ts:8]: Ready!

**Review by CrhistianT7 — APPROVED** [review]
LGTM!!

**Pull request merged**